src/main/resources/assets/arcanuscontinuum/lang/fr_fr.json

```
@@ -0,0 +1,334 @@
+	"item.arcanuscontinuum.battle_mage_boots_exposed": "Bottes de Mage de Guerre Exposées",
+	"item.arcanuscontinuum.battle_mage_helmet_weathered": "Casque de Mage de Guerre Erodé",
+	"item.arcanuscontinuum.battle_mage_chestplate_weathered": "Plastron de Mage de Guerre Erodé",
+	"item.arcanuscontinuum.battle_mage_leggings_weathered": "Jambières de Mage de Guerre Erodées",
+	"item.arcanuscontinuum.battle_mage_boots_weathered": "Bottes de Mage de Guerre Erodées",
+	"item.arcanuscontinuum.battle_mage_helmet_oxidized": "Casque de Mage de Guerre Oxydé",
+	"item.arcanuscontinuum.battle_mage_chestplate_oxidized": "Plastron de Mage de Guerre Oxydé",
+	"item.arcanuscontinuum.battle_mage_leggings_oxidized": "Jambières de Mage de Guerre Oxydées",
+	"item.arcanuscontinuum.battle_mage_boots_oxidized": "Bottes de Mage de Guerre Oxydées",
+	"item.arcanuscontinuum.battle_mage_helmet_waxed_unaffected": "Casque de Mage de Guerre Ciré",
+	"item.arcanuscontinuum.battle_mage_chestplate_waxed_unaffected": "Plastron de Mage de Guerre Ciré",
+	"item.arcanuscontinuum.battle_mage_leggings_waxed_unaffected": "Jambières de Mage de Guerre Cirées",
+	"item.arcanuscontinuum.battle_mage_boots_waxed_unaffected": "Bottes de Mage de Guerre Cirées",
+	"item.arcanuscontinuum.battle_mage_helmet_waxed_exposed": "Casque de Mage de Guerre Erodé Ciré",
+	"item.arcanuscontinuum.battle_mage_chestplate_waxed_exposed": "Plastron de Mage de Guerre Erodé Ciré",
+	"item.arcanuscontinuum.battle_mage_leggings_waxed_exposed": "Jambières de Mage de Guerre Erodées Cirées",
+	"item.arcanuscontinuum.battle_mage_boots_waxed_exposed": "Bottes de Mage de Guerre Erodées Cirées",
+	"item.arcanuscontinuum.battle_mage_helmet_waxed_weathered": "Casque de Mage de Guerre Erodé Ciré",
+	"item.arcanuscontinuum.battle_mage_chestplate_waxed_weathered": "Plastron de Mage de Guerre Erodé Ciré",
+	"item.arcanuscontinuum.battle_mage_leggings_waxed_weathered": "Jambières de Mage de Guerre Erodées Cirées",
+	"item.arcanuscontinuum.battle_mage_boots_waxed_weathered": "Bottes de Mage de Guerre Erodées Cirées",
+	"item.arcanuscontinuum.battle_mage_helmet_waxed_oxidized": "Casque de Mage de Guerre Oxydé Ciré",
+	"item.arcanuscontinuum.battle_mage_chestplate_waxed_oxidized": "Plastron de Mage de Guerre Oxydé Ciré",
+	"item.arcanuscontinuum.battle_mage_leggings_waxed_oxidized": "Jambières de Mage de Guerre Oxydées Ciré",
+	"item.arcanuscontinuum.battle_mage_boots_waxed_oxidized": "Bottes de Mage de Guerre Oxydées Ciré",
+	"item.arcanuscontinuum.battle_mage_upgrade_smithing_template": "Modèle de Forge",
+	"upgrade.arcanuscontinuum.battle_mage_upgrade": "Amélioration en Mage de Guerre",
+	"item.arcanuscontinuum.smithing_template.battle_mage_upgrade.applies_to": "Armure en Diamand",
+	"item.arcanuscontinuum.smithing_template.battle_mage_upgrade.ingredients": "Éclats d'Améthyste",
+	"item.arcanuscontinuum.smithing_template.battle_mage_upgrade.base_slot_description": "Ajouter une Armure en Diamand",
```

```suggestion
	"item.arcanuscontinuum.smithing_template.battle_mage_upgrade.base_slot_description": "Ajouter une armure en diamant",
```
"diamond" is written "diamant" in French, with a "t" instead of a "d"